atom/browser/api/atom_api_power_monitor.cc

```diff
@@ -68,10 +110,6 @@ using atom::api::PowerMonitor;
 
 void Initialize(v8::Local<v8::Object> exports, v8::Local<v8::Value> unused,
                 v8::Local<v8::Context> context, void* priv) {
-#if defined(OS_MACOSX)
-  base::PowerMonitorDeviceSource::AllocateSystemIOPorts();
```

Comment:
per comment suggestion in https://github.com/electron/electron/pull/11807#issuecomment-362507906 .